x-pack/test/accessibility/apps/transform.ts

```diff
@@ -147,7 +147,7 @@ export default function ({ getService }: FtrProviderContext) {
           await a11y.testAppSnapshot();
         });
 
-        it('create pivot transform configuration step JSON editor', async () => {
+        it.skip('create pivot transform configuration step JSON editor', async () => {
```

Comment:
this test was crashing the browser instance in CI, presumably because a `CodeEditor` instance (powered by monaco) consumes too much memory. Locally it was ok on my machine though.